README.md

```diff
@@ -1,57 +1,11 @@
-# tw-mkdocs
+# TW Profile in a Static Site
 
-## Installation
+This repository contains the code for the [static site](https://vira-wizeline.github.io/tw-profile/page-two/) about Vira Ivanova, who developed it as the [Final Assignment](https://docs.google.com/document/d/1Tz_xG8JD6K1xX7LNZpY28QPRyPO_D6FdX_PmZ16XzW0/edit) for the [**DocOps**](https://wizeline.atlassian.net/wiki/spaces/WTW/pages/2064548146/The+DocOps+Task+Force+Process) training. The information on this site reflects Vira's experience as a Technical Writer (TW).
 
-1. Install docker desktop from https://hub.docker.com/editions/community/docker-ce-desktop-mac
-2. Change directory to the tw-mkdocs folder
-```
-cd path/to/project/tw-docops/tw-mkdocs
-```
-3. Build mkdocs image.
+Vira used the following tools to create the site:
+- [**Markdown**](https://www.markdownguide.org/): To write the pages.
+- [**VSCode**](https://code.visualstudio.com/): To edit the content of the pages.
+- [**GitHub**](https://github.com/): To use versioning.
+- [**GitHub Pages**](https://pages.github.com/): To deploy the site.
 
-Execute the following command in the terminal using the current directory
-```
-docker build -t tw-mkdocs-img .
-```
-4. Serve your local environment.
-
-Execute the following command in the terminal after the docker image creation.
-```
-docker run --name vira-tw-profile -d -p 9090:9090 --volume="$PWD:/app" tw-mkdocs-img:latest
-```
-
->**Note:** Use the `-d` flag in the above command to detach your terminal from the running container.
-
-5. Start working. Open http://localhost:9090 to see all your changes.
-
-## Stop mkdocs
-
-Open docker's dashboard
-
-<img src="./docs/images/docker_dashboard.png" alt="Open docker's dashboard" width="250">
-
-
-Locate the wizeline-mkdocs-template container and stop it
-
-<img src="./docs/images/docker_stop.png" alt="Stop Docker" width="650">
-
-Wait until the terminal process stops
-
-## Restart mkdocs
-
-Open docker's dashboard
-
-<img src="./docs/images/docker_dashboard.png" alt="Open docker's dashboard" width="250">
-
-
-Locate the wizeline-mkdocs-template container and start it
-
-<img src="./docs/images/docker_restart.png" alt="Restart Docker" width="650">
-
-## Create static site
-1. Change directory to the tw-mkdocs folder
-
-2. Execute the following command
-```
-docker run --rm --volume="$PWD:/app" -it tw-mkdocs-img:latest mkdocs build
-```
+Here is another example of a [TW profile by Eduardo Ocejo](https://vira-wizeline.github.io/tw-profile/).
```

Comment:
Aww :3  I thought we were joking with this line hahahah ^_^
I feel obligated to mention that the URL is pointing to your site.

Reply:
My reply